recipes/win-unicode-console/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
This file should be removed. If the LICENSE is in the tarball, it shouldn't be in the recipe folder

Reply:
license file is not available in the source package for win_unicode_console https://files.pythonhosted.org/packages/89/8d/7aad74930380c8972ab282304a2ff45f3d4927108bb6693cabcc9fc6a099/win_unicode_console-0.5.zip